### PR TITLE
OSX license update

### PR DESCRIPTION
**Pull request by rasberrypai**
Updated license, welcome and conclusion pages of macOSX NSIS-themed installer.

### CODE REVIEW

File: installer/OSX/Installer/Resources/conclusion.html [hunk 1]
```diff
@@ -6,33 +6,22 @@
     </head>
     <body>
         <div style="font-family: Helvetica; padding-left: 10px;" align="left">
-            <h3>__PRODUCT__ : __VERSION__</h3>
-                <p style="color: #020202; font-size: 11px;">Thank you for installing Synthesis: An Autodesk Application</p>
-            <h5>Run __PRODUCT__</h5>
-                <p style="color: #020202; font-size: 11px;">Open your quick launch browser using Command+Space:</p>
-                <code style="color: #c6a72b; font-size: 11px;"><span style="color: #abb0b0">__PRODUCT__</span></code>
+            <h3>Synthesis: An Autodesk Application | 4.3.3.1</h3>
+                <p style="color: #020202; font-size: 12px;">Thank you for installing Synthesis: An Autodesk Application</p>
```
Comment (matthew17754) [inline installer/OSX/Installer/Resources/conclusion.html:10]: Synthesis: An Autodesk Technology

File: installer/OSX/Installer/Resources/welcome.html [hunk 2]
```diff
@@ -5,9 +5,7 @@
 </head>
 <body>
     <div align="left" style="font-family: Helvetica; padding-left: 10px;">
-    	<br/>
-        <p style="color: #020202; font-size: 12px;">This will install <span style="color: #FF7F00; font-size: 12px;">Synthesis: An Autodesk Application __VERSION__</span> on your computer. You will be guided through the steps necessary to install this software.</p>
-        <br/>
+        <p style="color: #020202; font-size: 12px;">This will install <span style="color: #FF7F00; font-size: 12px;">Synthesis: An Autodesk Application 4.3.3.1</span> on your computer. You will be guided through the steps necessary to install this software.</p>
```
Comment (matthew17754) [inline installer/OSX/Installer/Resources/welcome.html:8]: Synthesis: An Autodesk Technology

File: installer/OSX/Installer/Resources/license.html [hunk 3]
```diff
@@ -185,16 +185,7 @@
...
-boilerplate notice, with the fields enclosed by brackets "[]"
-replaced with your own identifying information. (Don't include
-the brackets!)  The text should be enclosed in the appropriate
-comment syntax for the file format. We also recommend that a
-file or class name and description of purpose be included on the
-same "printed page" as the copyright notice for easier
-identification within third-party archives.
-<br><br><br>
-Copyright [2019] [Autodesk inc.]
+Copyright 2020 Autodesk inc.
```
Comment (matthew17754) [inline installer/OSX/Installer/Resources/license.html:188]: Copyright 2020 Autodesk Inc.

File: installer/OSX/Installer/Resources/conclusion.html [hunk 4]
```diff
@@ -6,33 +6,22 @@
...
         <div style="font-family: Helvetica; padding-left: 10px;" align="left">
             <br/>
-            <h5>Uninstall __PRODUCT__</h5>
-                <p style="color: #020202; font-size: 11px;">Run the following command to uninstall __PRODUCT__<br />
-                  <code style="color: #c6a72b; font-size: 11px;"><br />&nbsp; $ <span style="color: #abb0b0">sudo bash ~/Library/Application\ Support/__PRODUCT__/__VERSION__/uninstall.sh</span></code>
-                </p>
-        </div>
-        <div style="font-family: Helvetica; padding-left: 10px;" align="left"><br />
-            <p style="color: #abb0b0; font-size: 10px;">Copyright © 2019 Autodesk inc.</p>
+	    <p style="color: #abb0b0; font-size: 10px;">Copyright © 2020 Autodesk inc.</p>
```
Comment (matthew17754) [inline installer/OSX/Installer/Resources/conclusion.html:24]: Copyright 2020 Autodesk Inc.

**Review by matthew17754 — CHANGES_REQUESTED** [review]
Some minor branding technicalities

**Review by matthew17754 — APPROVED** [review]

**Review by KiraCorbett — APPROVED** [review]

**Pull request merged**